### PR TITLE
adjust ceos-lab interface type and import topology defined environment variables into container

### PR DESCRIPTION
**Pull request by sulrich**
currently, to provide better alignment with some features internal to ceos-lab, interfaces should be named `et` and not `eth`.    
this PR modifies the default ceos-lab invocations to provide this consistency. 

- updates relevant elements of clab/ceos.go
- updates manual/kinds/ceos.md to align with the env changes and outlines the use in topologies.

### CODE REVIEW

File: docs/manual/kinds/ceos.md [hunk 1]
```diff
@@ -39,13 +39,24 @@ Arista cEOS node launched with containerlab can be managed via the following int
...
+
 ## Interfaces mapping
 ceos container uses the following mapping for its linux interfaces:
 
 * `eth0` - management interface connected to the containerlab management network
-* `eth1` - first data interface
+* `et1` - first data interface
 
-When containerlab launches ceos node, it will set IPv4/6 addresses as assigned by docker to the `eth0` interface and ceos node will boot with that addresses configure. Data interfaces `eth1+` need to be configured with IP addressing manually.
+When containerlab launches ceos node, it will set IPv4/6 addresses as assigned by docker to the `et0` interface and ceos node will boot with that addresses configure. Data interfaces `et1+` need to be configured with IP addressing manually.
```
Comment (hellt) [inline docs/manual/kinds/ceos.md:59]: since `eth0` interface is not part of this PR, it shouldn't have changed here in this sentence I suppose?

**Comment by hellt** [comment]
Hi @sulrich 
thanks for the contribution!
Is my understanding correct, that the translation from `ethX` to `etX` will be carried out by ceos? Or does containerlab need to rename the interfaces `ethX` to `etX`?

File: docs/manual/kinds/ceos.md (hunk 1 above)
Comment (sulrich) [reply docs/manual/kinds/ceos.md:59]: ah! good catch.  i'll revise this.

**Comment by sulrich** [comment]
> Hi @sulrich
> thanks for the contribution!
> Is my understanding correct, that the translation from `ethX` to `etX` will be carried out by ceos? Or does containerlab need to rename the interfaces `ethX` to `etX`?

the interfaces that are created by containerlab should be renamed to `etX`. this provides the necessary alignment for certain things to work consistently. (notably some interactions w/i IGPs and link-local functions)  this will likely go away in the future.  but setting `$INTFTYPE` to `et` provides the necessary alignment to make folks happy. 

i didn’t include `eth0` in this as the interactions aren’t commonly over the `Ma0` interface and this is largely outside the scope of what containerlab users would manipulate anyway.

File: docs/manual/kinds/ceos.md (hunk 1 above)
Comment (sulrich) [reply docs/manual/kinds/ceos.md]: updated text accordingly.

**Comment by sulrich** [comment]
@hellt just to follow up on misc. discussion.  this also addresses the merged
environment for invoking the `node.Cmd`. while the environment for docker was
being set. the fixed `node.Cmd` did not import the environment variables
defined in the topology file into the actual container. 

this also ensures that the environment variables defined in the topology file
are pushed into the container.

**Comment by sulrich** [comment]
changing approach here - closing in lieu of different PR
